app/models/miq_schedule_worker/jobs.rb

```diff
@@ -117,6 +117,10 @@ def policy_event_purge_timer
     queue_work(:class_name => "PolicyEvent", :method_name => "purge_timer", :zone => nil)
   end
 
+  def miq_report_result_purge_timer
+    queue_work(:class_name => "MiqReportResult", :method_name => "purge_timer", :zone => nil)
```

Comment:
* nil zone
* schedules_for_scheduler_role

This means the singular regional "scheduler" role schedule worker will run this and queue it for the deletion of old report results for the whole region.   If you were concerned with who should run it, I think the above confirms you did it right.